src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
```

Comment:
Consider using a more descriptive variable name than 'population' to clarify that it is a collection of elements, such as 'populationElements'.